### PR TITLE
Update master fork with updates

### PR DESCRIPTION
**Pull request by 1337-server**
# Description

Large upgrade to bring the main master branch in line with current dev environment, this should solve a number of issues where file changes/moves over versions are causing issues

This will also update the issue/PR templates to be updated to the newer GitHub versions.

## Fixes
 N/A Too many to list

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration

- [x] Ubuntu 18.04
- [x] Ubuntu 20.04
- [x] Debian 10
- [x] Debian 11
- [x] Docker
- [ ] Other (Please state here)

# Checklist:

- [ ] **I have submitted this PR against the `v2_devel` branch (Unless the main branch has a security issue)**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have tested that my fix is effective or that my feature works

### CODE REVIEW

**Comment by shitwolfymakes** [comment]
We need to make sure that the updated issue template has checkbox for users to confirm that they have checked the wiki to confirm that they've installed ARM on a supported OS. 

Lot of users not checking it and submitting issues caused by installation on Ubuntu 22.04

**Comment by 1337-server** [comment]
> We need to make sure that the updated issue template has checkbox for users to confirm that they have checked the wiki to confirm that they've installed ARM on a supported OS.
> 
> Lot of users not checking it and submitting issues caused by installation on Ubuntu 22.04

This is already in the new form

https://github.com/automatic-ripping-machine/automatic-ripping-machine/blob/v2_devel/.github/ISSUE_TEMPLATE/bug_report.yaml

**Comment by shitwolfymakes** [comment]
Ah, thanks!

**Comment by 1337-server** [comment]
@shitwolfymakes This is ready when you are.

Not sure if this even needs to be reviewed, as it's just the updates from the dev branch.

File: arm/ui/routes.py [hunk 1]
```diff
@@ -418,56 +487,26 @@
...
-                        while True:
-                            yield f.read()
-                            sleep(1)
-                except Exception:
-                    return render_template('simple_error.html')
+        generate = ui_utils.generate_full_log(full_path)
     elif mode == "download":
-        app.logger.debug('fullpath: ' + fullpath)
-        return send_file(fullpath, as_attachment=True)
+        return send_file(full_path, as_attachment=True)
```
[the remark this answers is not in the document]
Comment (shitwolfymakes) [reply arm/ui/routes.py:341]: Risk accepted on the basis that this should NEVER face the public internet

**Comment by shitwolfymakes** [comment]
Will test on as many systems as I can manage tomorrow

**Comment by shitwolfymakes** [comment]
I'll review this tomorrow

**Review by shitwolfymakes — APPROVED** [review]

**Pull request merged**